src/core/Window.cpp

```diff
@@ -454,13 +454,13 @@ void Window::InitializeLogging() {
 #ifndef __WIIU__
         auto logPath = GetPathRelativeToAppDirectory(("logs/" + GetName() + ".log").c_str());
         auto fileSink = std::make_shared<spdlog::sinks::rotating_file_sink_mt>(logPath, 1024 * 1024 * 10, 10);
-        fileSink->set_level(spdlog::level::trace);
+        fileSink->set_level(spdlog::level::debug);
```

Comment:
Should ifdef here for `_DEBUG`. If it's `_DEBUG`, allow `trace` level, otherwise `debug` level.